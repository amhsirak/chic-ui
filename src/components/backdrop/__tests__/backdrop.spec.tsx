import Backdrop from '..';
import { render } from '../../../test/utils';

/* eslint-disable */
describe('Backdrop Component', () => {
  it('Renders Correctly', () => {
    const BackdropGroup = render(
      <>
        <Backdrop />
        <Backdrop color="#e4f0f4" />
        <Backdrop visible={false} />
      </>
    );
    expect(BackdropGroup).toMatchSnapshot();
  });
});
