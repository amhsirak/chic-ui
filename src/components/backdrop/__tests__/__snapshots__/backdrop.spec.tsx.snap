// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backdrop Component Renders Correctly 1`] = `
Array [
  <div
    className="sc-bdfBQB fiZSsf"
  />,
  <div
    className="sc-bdfBQB gqAeyi"
    color="#e4f0f4"
  />,
  <div
    className="sc-bdfBQB gzdGhb"
  />,
]
`;
